devel/mps_load_test.sh

```diff
@@ -208,6 +202,7 @@ cd "$src/devel"
 
 # Build a version of mod_pagespeed with all optimizations enabled, but with
 # a build that includes DCHECKs.
+# TODO(oschaaf): can we skip this step if we have a custom .so?
```

Comment:
It sets up the config as well, so you would want to at least use apache_trace_stress_test_prepare